todo_app/app.py

```diff
@@ -1,11 +1,53 @@
-from flask import Flask
+#!
+from flask import Flask, appcontext_popped, render_template, redirect, request
+from flask_config import Config
+from dotenv import load_dotenv, find_dotenv
+from data.trello_items import *
 
-from todo_app.flask_config import Config
+def create_app():
 
-app = Flask(__name__)
-app.config.from_object(Config())
+   file_path = find_dotenv('.env')
+   load_dotenv(file_path)
 
+   token = os.getenv('TRELLO_TOKEN')
+   key = os.getenv('TRELLO_KEY')
+   board_ID = os.getenv('BOARD_ID')
+   url = "https://api.trello.com/1/"
+   url_member = url + "members/carispang"
+   query_string = {"key" : key, "token" : token}
+   
+   app = Flask(__name__)
+   app = Flask(__name__, template_folder='templates')
+   app.config.from_object(Config())
 
-@app.route('/')
-def index():
-    return 'Hello World!'
+   @app.route('/', methods = ['GET', 'POST'])
+   def createUI():
+      data_board_cards = get_data_board_cards(url, query_string)
+      data_board_lists = get_board_lists(url, query_string, board_ID)
+      list_id = get_list('To Do', data_board_lists)
+      todo_cards = get_cards_on_list(url, list_id, query_string)
+      if request.method == 'POST':
+         if not request.form.get('complete_button'):
+            card_name = request.form.get('submit_button') 
+            create_card(url, list_id, card_name, key, token)
+            todo_cards.append({"name": card_name})
+         else:
+            data_board_lists = get_board_lists(url, query_string)
+            done_ID = get_list('Done', data_board_lists)
+            complete_method(data_board_cards, url, done_ID, key, token)
+            todo_cards = get_cards_on_list(url, list_id, query_string)
+      return render_template('index.html', ls = todo_cards)
+
+   @app.route('/ExerciseFive')
+   def exercise_five():
+      data_board_cards = get_data_board_cards(url, query_string)
+      card_info = get_card_info(data_board_cards)
+      list_info = get_list_info(data_board_cards)
+      new_list = get_listForLoop(card_info, list_info)
+      cardNames = get_card_names(data_board_cards)
+    
+      return render_template('index_3.html', ls = new_list)
+
+   app.run()
+
+create_app()
```

Comment:
This doesn't work for me doing `poetry run flask run` in the root of your repository. Flask expects `create_app` to return an app object, i.e.

```suggestion
   return app
```

If you want to be able to run this file directly with `poetry run python todo_app/app.py` then you could do
```suggestion
   return app

if __name__ == "__main__":
   create_app().run()
```